torch/_prims_common/wrappers.py

```diff
@@ -224,6 +224,12 @@ def _fn(*args, out=None, **kwargs):
                     if k not in kwargs:
                         kwargs[k] = out_attr
 
+            # out_names can sometimes appear in kwargs. But Python metas are
+            # defined without these params (hence this wrapper). This removes
+            # out_names from kwargs and sets their values as out.
+            if any(x in kwargs for x in out_names):
+                out = tuple(kwargs.pop(x) for x in out_names)
+
```

Comment:
- Fixes `test_dispatch_meta_outplace_linalg_eigvalsh_cpu_float64` (see the other comment)
- Partially fixes `test_dispatch_symbolic_meta_outplace_all_strides_msort_cuda_float32` (fully fixed in the next PR in the stack).

Haven't looked deep, but both of the mentioned tests are `CompositeImplicitAutograd` funcs. CPP impl ends up calling a Python meta via the dispatcher. So maybe something related to the way the dispatcher passes args around. Maybe this: https://github.com/pytorch/pytorch/pull/79941#discussion_r907818660